docs/source/format/DissociatedIPC/SequenceDiagramSeparate.mmd

```diff
@@ -0,0 +1,47 @@
+%% Licensed to the Apache Software Foundation (ASF) under one
+%% or more contributor license agreements.  See the NOTICE file
+%% distributed with this work for additional information
+%% regarding copyright ownership.  The ASF licenses this file
+%% to you under the Apache License, Version 2.0 (the
+%% "License"); you may not use this file except in compliance
+%% with the License.  You may obtain a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+
+%% To generate the diagram, use mermaid-cli
+%% Example: docker run --rm -v $(pwd)/DissociatedIPC:/data minlag/mermaid-cli -i /data/SequenceDiagramSeparate.mmd
+
+sequenceDiagram
+  participant D as Data Stream
+  participant C as Client
+  participant M as Metadata Stream
+
+  activate C
+  C-->>+M: TAG: <want_data> BODY: <opaque ID>
+  C-->>+D: TAG: <want_data> BODY: <opaque ID>
+  M-->>C: MSG: [1] + <sequence number> + <schema metadata>
+  loop each chunk
+    par
+      M-->>C: MSG: [1] + <sequence number> + <IPC metadata>
+    and
+      alt
+        D-->>C: TAG: <inline data> | <sequence number> BODY: <raw body bytes>
```

Comment:
I'm not sure I understand the notation here. If I look at the description of tags at the same time I can read this as "the tag type is Tag::INLINE_DATA (which requires raw body bytes), and it includes a sequence number (which all tags include... except those sent by the consumer?)". I think this could be more understandable if it looked just a bit more formal:

```
  C-->>+M: TaggedMessage(server.want_data, bytes=ID_of_desired_data)
  C-->>+D: TaggedMessage(server.want_data, bytes=ID_of_desired_data)
  M-->>C: Message(bytes([1]) + le_bytes(sequence_number) + schema_metadata)
  loop each batch
    par
      M-->>C: Message(bytes([1]) + le_bytes(sequence_number) + batch_metadata)
    and
      alt
        D-->>C: TaggedMessage(sequence_number, bytes=batch_data)
      else
        D-->>C: TaggedMessage(sequence_number, uint64_pairs=offset_size_list)
```

(also I think if it's correct to substitute batch for chunk that'd be more idiomatically Arrow)

Reply:
updated the diagrams, let me know what you think